### DISCUSSION TITLE
Disable all dropping in Editor

### DISCUSSION THREAD

**Discussion by GalGurGurevich** [GrapesJS/grapesjs#4396] [Q&A]
Hi to all,
I'm trying to prevent ALL dropping into my editor . in the editor the default allows to drag and drop elements even from a different place thus can lead to a security issue.

is there a way to disable all drop into GrapesJS? 

I tried using 
droppable: false,
draggable: false,

on all elements yet you can still drag&drop from outside the canvas inside.

**Comment by GalGurGurevich** [comment]
nevermind fixed it with: 
editor.getWrapper().set({
        'droppable': false
});

**Reply by ronaldohoch** [reply]
Hello, how do you drop the blocks from grapes with this config?

**Comment by ducvu91** [comment]
> nevermind fixed it with: editor.getWrapper().set({ 'droppable': false });

Is run ? not working for me :(
